src/etc/inc/dyndns.class

```diff
@@ -219,6 +223,8 @@
 			switch ($dnsService) {
 			case 'freedns':
 			case 'freedns-v6':
+			case 'freedns-v2':
+			case 'freedns-v6-v2':
```

Comment:
Using `-v6-v2` seems confusing... Maybe this should be `freedns2` and `freedns2-v6`?

Reply:
Of course, that's way better.